site/src/book/filtering-expression.md

```diff
@@ -0,0 +1,47 @@
+# Filtering expression
+
+* **Introduced in:** cargo-nextest 0.9.13
+* **Environment variable**: `NEXTEST_EXPERIMENTAL_EXPR_FILTER=1`
+* **Tracking issue**: []
+
+Tests to run can be filtered using filter expressions.
+
+## Filtering expression DSL
+
+A filtering expression define a set of tests, any test in the set will be run.
+
+Basic sets:
+- `all()`: include everything
+- `test(name-matcher)`: include all tests matching `name-matcher`
+- `package(name-matcher)`: include all tests in packages matching `name-matcher`
+- `deps(name-matcher)`: include all tests in packages depended by packages matching `name-matcher` and include all tests in packages matching `name-matcher`
+- `rdeps(name-matcher)`: include all tests in packages depending on packages matching `name-matcher` and include all tests in packages matching `name-matcher`
+- `none()`: include nothing
```

Comment:
Could you add a `partition` operator as well?

Reply:
Like `--partition`, this might be a little bit more complex and requires state.

Can we do it after, in an other PR ?